src/test_typing_extensions.py

```diff
@@ -1872,18 +1872,6 @@ class E(C, BP): pass
         self.assertNotIsInstance(D(), E)
         self.assertNotIsInstance(E(), D)
 
-    def test_runtimecheckable_on_typing_dot_Protocol(self):
-        @runtime_checkable
-        class Foo(typing.Protocol):
-            x: int
-
-        class Bar:
-            def __init__(self):
-                self.x = 42
-
-        self.assertIsInstance(Bar(), Foo)
-        self.assertNotIsInstance(object(), Foo)
-
```

Comment:
We can't support this anymore, now that we once again reimplement `runtime_checkable` on Python <=3.12. We never documented that you could do this, though; it was just a nice-to-have-if-possible.